scripts/deploy-cloud-components.sh

```diff
@@ -23,6 +23,10 @@ kubectl apply -f ./tmp/openfaas-cloud/yaml/core/edge-router-svc.yml
 
 kubectl apply -f ./tmp/openfaas-cloud/yaml/core/edge-auth-svc.yml
 
+if [ "$ENABLE_NETWORK_POLICIES" = "true" ] ; then
+    kubectl apply -f ./tmp/openfaas-cloud/yaml/network-policy/ns-openfaas-fn-net-policy.yml
```

Comment:
We may rename the files, therefore can we just apply that folder instead of specific files?

Reply:
Yeah of course I saw the `edge-atuh-svc.yml` and followed the example, but I will re-do this 👍